pages/common/git-extras.md

```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Set of commands for `Git`.
+> More information: <https://github.com/tj/git-extras>.
+
+- Download and or upgrade `git-extras` commands:
```

Comment:
```suggestion
- Download and or upgrade `git-extras` commands:
```

This is correct Engrish to my knowledge however what do you think

Reply:
That suggestion doesn't change anything, @CleanMachine1.

Reply:
Also, I've never (to my knowledge) heard or spoke Engrish, can you enlighten me?

Reply:
It was ironic.. 
i didn't mean to use the suggestion, muscle memory.

is the `and or` acceptable

Reply:
"and" followed by "or" is perfectly fine, but I'd prefer a solidus in between:
```suggestion
- Download and/or upgrade `git-extras` commands:
```

Reply:
14 files have it, committed.